pallets/protos/src/lib.rs

```diff
@@ -170,7 +170,7 @@ pub mod pallet {
 
 	/// **StorageMap** that maps a **Trait ID** to the **name of the Trait**
 	#[pallet::storage]
-	pub type Traits<T: Config> = StorageMap<_, Identity, Hash64, Vec<u8>, ValueQuery>;
+	pub type Traits<T: Config> = StorageMap<_, Identity, Hash64, Vec<u8>, OptionQuery>;
```

Comment:
what does `OptionQuery` do as opposed to `ValueQuery` btw?

Reply:
`get` returns `Option<_>` instead of default